### PR TITLE
improved StateDebugView

### PR DESCRIPTION
**Pull request by sqs**
- more minimal UI so you can keep it enabled while devving
- shows in Cody Web demo

image

## Test plan

n/a

### CODE REVIEW

File: web/lib/agent/agent.client.ts [hunk 1]
```diff
@@ -85,6 +85,7 @@ export async function createAgentClient({
                 'cody.autocomplete.enabled': false,
                 'cody.experimental.urlContext': true,
                 'cody.web': true,
+                'cody.internal.debug.state': true,
```
Comment (dominiccooney) [inline web/lib/agent/agent.client.ts:88]: Would this mean web is seeing this debug state by default? CC @vovakulikov

Comment (sqs) [reply web/lib/agent/agent.client.ts:88]: Only in the internal `pnpm -C web dev` subproject. Not in the Cody Web product.

Comment (sqs) [reply web/lib/agent/agent.client.ts:88]: Actually, nevermind...it would. Will change.

**Review by pkukielka — APPROVED** [review]
LGTM

**Pull request merged**